amr-wind/wind_energy/actuator/disk/constant_ct_ops.cpp

```diff
@@ -0,0 +1,128 @@
+#include "amr-wind/wind_energy/actuator/disk/constant_ct_ops.H"
+namespace amr_wind {
+namespace actuator {
+namespace ops {
+void collect_parse_conflicts(
+    const utils::ActParser& pp,
+    const std::string& p1,
+    const std::string& p2,
+    std::ostringstream& ss)
+{
+    if (pp.contains(p1) && pp.contains(p2)) {
+        ss << "ConstantCt Conflict: " << p1 << " and " << p2 << std::endl;
+    }
+}
+void collect_parse_dependencies(
+    const utils::ActParser& pp,
+    const std::string& p1,
+    const std::string& p2,
+    std::ostringstream& ss)
+{
+    if (pp.contains(p1) && !pp.contains(p2))
+        ss << "ConstantCt Dependency Missing: " << p1 << " and " << p2
+           << std::endl;
+    if (!pp.contains(p1) && !pp.contains(p2))
+        ss << "ConstantCt Dependency Missing: " << p1 << " and " << p2
+           << std::endl;
+}
+
+void required_parameters(ConstantCt::MetaType& meta, const utils::ActParser& pp)
+{
+    pp.get("num_force_points", meta.num_force_pts);
+    pp.get("disk_center", meta.center);
+    pp.get("epsilon", meta.epsilon);
+    pp.get("rotor_diameter", meta.diameter);
+    pp.get("thrust_coeff", meta.thrust_coeff);
```

Comment:
I am wondering if it would be best to have a table with velocity and ct such that the thrust coefficient can change according to velocity. If only one ct is given then it would default to constant ct.

Reply:
@tonyinme sure that is a good idea.

Reply:
@tonyinme I'm going to add this in a follow on PR.  This one has gotten pretty big.